### DISCUSSION TITLE
Prevent spinning up actions during release with multiple simultaneous pushes across branches

### DISCUSSION THREAD

**Discussion by thornomad** [community/community#26500] [Actions]
<p>Using git-flow, when we finish and release a feature it ends up making three separate pushes to origin.  One on master, one on develop, and one for the tag.  This results in three actions getting spun up, one for each of them.</p>
<p>Is there any way to prevent this, in this special use case, and only run one action (instead of three? </p>
<p>I know I can set the <code>on</code> variable to limit it to a branch like master; however, we do <em>want</em> to be able to trigger builds when things are pushed independently (for example, when a feature branch is pushed). </p>
<p>Curious if anyone has any insight or possible solutions.  Just trying to save some CPU time.</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/thornomad">@thornomad</a> ,</p>
<p>It’s not supported. For push event, it only concerns whether there’s a commit while not commit created by gitflow or direct git push.  If the yaml file is identical between the three branches, you can limit the workflow to a specific branch, but it will prevent individual push workflow of the other branches.</p>
